solution/ui/regulations/css/scss/partials/_site_homepage.scss

```diff
@@ -195,12 +210,17 @@
                 padding-right: 25px;
                 margin-bottom: 16px;
 
-                .v-tabs-bar {
+                .v-slide-group__container {
```

Comment:
Vuetify 3 refactored several of their components to have different classes and be comprised of different elements when compared to Vuetify 2; most `v-` prefixed changes are the response to this change.